src/style.css

```diff
@@ -0,0 +1,22 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.red-box {
+  background-color: red;
+  width: auto;
+  height: 100px;
```

Comment:
create a common class, which has this style and use it everywhere you need this style